pkg/ovs/ovs.go

```diff
@@ -40,18 +25,24 @@ func (tx *Transaction) exec(cmd string, args ...string) (string, error) {
 		return "", tx.err
 	}
 
+	cmdpath, err := exec.LookPath(cmd)
```

Comment:
We are following this pattern:
  Start Transaction
  Bunch of operations (calls exec to perform the op)
  End Transaction
Instead of doing LookPath(cmd) for every exec(), another option is to set vsctlPath and ofctlPath in NewTransaction() if not initialized. Any operation is performed using transaction object, so this will ensure vsctl/ofctl paths always set.